Galaxy.json

```diff
@@ -1753,6 +1754,6 @@
   "timezone": "browser",
   "title": "Galaxy",
   "uid": "000000004",
-  "version": 60,
+  "version": 9,
```

Comment:
this looks wrong, should that be increased?

Reply:
I had the doubt as well and wasn't sure. The previous PRs I made did not change the version. So, don't know.

Reply:
from the docs `version of the dashboard (integer), incremented each time the dashboard is updated`

Reply:
I will add a commit updating the number to `61` instead.